scripts/environment-provisioning/aws/eks/set-terraform-variables.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+ARGS="$*"
+"/bin/sh" "../../../set-config.sh" "$ARGS"
```

Comment:
```suggestion
../../../set-config.sh "$ARGS"
```

The problem is with variables.

set-config.sh should be executable. Please check and fix if needed.